package.nls.fr.json

```diff
@@ -0,0 +1,25 @@
+{
+  "command.regenerateIcons.title": "Appliquer Icônes Personnalisation",
+  "command.ngPreset.title": "Basculer Angular Préréglé (Niveau d'espace de travail)",
+  "command.jsPreset.title": "Basculer JS Officiel Préréglé (Niveau de l'utilisateur)",
+  "command.tsPreset.title": "Basculer TS Officiel Préréglé (Niveau de l'utilisateur)",
+  "command.jsonPreset.title": "Basculer JSON Officiel Préréglé (Niveau de l'utilisateur)",
```

Comment:
These `toggle preset` translations also sound a little strange to me. I miss the article. If it's ok to you I'll ask one of my work colleagues to review these translations and will come back to you. He's from Belgium.